.github/workflows/deploy.yml

```
@@ -1,46 +1,31 @@
         with:
-          python-version: '3.11'
-
-      - name: Export gcloud related env variable
-        run: export CLOUDSDK_PYTHON="/usr/bin/python3"
+          credentials_json: ${{ secrets.GCP_SA_KEY_JSON }}
 
-      # Build and push image to Google Container Registry
-      - name: Setting up
-        uses: google-github-actions/setup-gcloud@v0
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
```

actionlint (1.7.7)

the runner of "google-github-actions/setup-gcloud@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)